examples/ucp_client_server.c

```diff
@@ -580,13 +580,10 @@ static int send_recv_am(ucp_worker_h ucp_worker, ucp_ep_h ep, int is_server,
  */
 static void ep_close(ucp_worker_h ucp_worker, ucp_ep_h ep)
 {
-    ucp_request_param_t param;
     ucs_status_t status;
     void *close_req;
 
-    param.op_attr_mask = UCP_OP_ATTR_FIELD_FLAGS;
-    param.flags        = UCP_EP_CLOSE_FLAG_FORCE;
-    close_req          = ucp_ep_close_nbx(ep, &param);
+    close_req = ucp_ep_close_nb(ep, UCP_EP_CLOSE_MODE_FORCE);
```

Comment:
why needed?
we prefer using NBX API instead

Reply:
I'll revert the change here since we prefer using NBX API.
The reason that I did this change is because there's below comment in this program and I want to align the code with the comment:
https://github.com/openucx/ucx/blob/v1.11.2/examples/ucp_client_server.c#L195
```
UCP_EP_CLOSE_MODE_FORCE which
```